packages/ddex/processor/src/publishRelease.ts

```diff
@@ -195,6 +195,36 @@ export function prepareTrackMetadatas(release: DDEXRelease) {
           sound.indirectContributors.map(mapContributor),
       }
 
+      if (release.deal?.audiusDealType == 'FollowGated') {
```

Comment:
I don't think this section is right
downloads and streams aren't necessarily the same gating.

For example, I should be able to specify deals like this
```
<Deal>
    <DealTerms>
        <CommercialModelType UserDefinedValue="FollowGated">UserDefined</CommercialModelType>
        <Usage>
            <UseType>PermanentDownload</UseType>
        </Usage>
        <TerritoryCode>Worldwide</TerritoryCode>
        <ValidityPeriod>
            <StartDate>2023-09-02</StartDate>
        </ValidityPeriod>
    </DealTerms>
</Deal>
<Deal>
    <DealTerms>
        <CommercialModelType>FreeOfChargeModel</CommercialModelType>
        <Usage>
            <UseType>OnDemandStream</UseType>
        </Usage>
        <TerritoryCode>Worldwide</TerritoryCode>
        <ValidityPeriod>
            <StartDate>2023-09-02</StartDate>
        </ValidityPeriod>
    </DealTerms>
</Deal>
```

which should translate to no stream conditions, but yes download conditions.


https://kb.ddex.net/implementing-each-standard/best-practices-for-all-ddex-standards/deals-and-commercial-aspects/multiple-deals-for-one-release/

@michellebrier does that check out?

Reply:
yes correct as i understand it